test/locators.spec.ts

```diff
@@ -0,0 +1,42 @@
+import { browser } from 'protractor';
+import { PersonalInformationPage } from '../src/page';
+import { PersonalData } from '../src/personal-data';
+
+describe('Given a page with a form', async () => {
+  beforeAll(async () => {
+    await browser.get('https://www.tutorialspoint.com/selenium/selenium_automation_practice.htm');
+  });
+
+  describe('When want to fill the form', async () => {
+    const personalData = new PersonalData();
+    Object.assign(personalData, {
+      firstName: 'Alejandro',
+      lastName: 'Perdomo',
+      sex: 'Male',
+      experience: 7,
+      profession: ['Automation Tester'],
+      tools: ['Selenium Webdriver'],
+      continent: 'South America',
+      commands: [
+        'Browser Commands',
+        'Navigation Commands',
+        'Switch Commands',
+        'Wait Commands',
+        'WebElement Commands']
+    });
```

Comment:
maybe you can create the object without the class. What about to use interfaces instead?